jyulb_cpp/internal/common/test_domain.py

```diff
@@ -0,0 +1,93 @@
+import domain
```

Comment:
what is the purpose of this file? is it a test? if so, it should be made into a test. If it's an example, it should be under examples.